File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/python/java/p03813.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
1000

Expected output:
ABC

```
a = int(input())
print('ABC' if a<1200 else 'ARC')
```


Answer:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        String result = a < 1200? "ABC" : "ARC";
        System.out.println(result);
    }
}
```

You can run this code with the following test case:

```java
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class MainTest {
    @Test
    public void testMain() {
        Main.main(new String[]{"1000"});
    }
}
```<|endoftext|>

File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/java/python/p03813.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
1000

Expected output:
6BC

```
import java.util.*;
public class p03813{
	static Scanner s = new Scanner(System.in);
	public static void main(String[] args) {
		System.out.println(s.nextInt()<1200?"ABC":"ARC");
	}
}
```


Answer:
```python
import sys

def main():
    n = int(sys.stdin.readline())
    if n < 1200:
        print("ABC")
    else:
        print("ARC")

if __name__ == "__main__":
    main()
```

You can write the following test cases to ensure it works correctly:

```python
import io
import sys
from unittest.mock import patch

def test_main():
    test_inputs = [
        "1000",
    ]
    test_outputs = [
        "6BC",
    ]

    for test_input, test_output in zip(test_inputs, test_outputs):
        with patch("sys.stdin", io.StringIO(test_input)):
            with patch("sys.stdout", new_callable=io.StringIO) as fake_out:
                main()
                assert fake_out.getvalue() == test_output + "\n"

test_main()
```<|endoftext|>
